.skip-usdchecker-files

```diff
@@ -0,0 +1,11 @@
+test_assets/NormalsTextureBiasAndScale/NormalsTextureBiasAndScale.usdz
+test_assets/NormalsTextureBiasAndScale/NormalsTextureBiasAndScale.usda
+test_assets/foundation/stage_composition/subLayer/sublayer_invalid.usda
+test_assets/foundation/stage_composition/references/reference_invalid.usda
+test_assets/foundation/stage_composition/payload/payload_invalid.usda
+test_assets/foundation/stage_composition/references_prim/reference_prim_in_same_file.usda
+test_assets/foundation/stage_composition/references_prim/reference_prim_in_other_file.usda
+test_assets/foundation/stage_configuration/invalid_defaultPrim/invalid_defaultPrim.usda
+test_assets/foundation/stage_configuration/framesPerSecond/framesPerSecond_0.usda
+test_assets/foundation/stage_configuration/framesPerSecond/framesPerSecond_-1.usda
```

Comment:
Although usdchecker fails for these two assets, it does so by crashing in the parsing stage - implying a bug in USD. (Tested with USD v23.05.) Perhaps it would be best to report this as a bug to the OpenUSD repo, if it still persists in v23.08 (to be released today).

Regardless, I don't think these are intended failure cases.

Reply:
Okay, I'll test this in v23.08 and if it's still happening, report it there